catalog/ui/search-ui/simple/yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
❓ Should this file be added?

Reply:
Hmm wonder how that got in there. good catch